static/index.js

```diff
@@ -1,5 +1,11 @@
 window.onload = () => {
-    alert("Hello world!");
-  };
+  alert("Hello world!");
+};
 
-  
\ No newline at end of file
+const item1 = (document.createElement("li").innerText = "take garbage out");
+const item2 = (document.createElement("li").innerText = "Play video game");
+const item3 = (document.createElement("li").innerText = "Take over the world");
+
+const listItems = document.createElement("ul");
+listItems.append(item1,item2,item3);
+const vanillaList = document.getElementById("list").appendChild(listItems);
```

Comment:
Move this code into your onload function.

So the issue with your list items is you are attempting to create the element and immediately set a value on it hoping that just the element you created gets set to itemX, however what really happens is itemX gets set to the string you are setting to innerText.

```
const item = document.createelement('li');
item.innerText = 'Hello World';
```

Reply:
Moving the code into the onload function gives you the ability to have your page render completely before running your javascript, since your code so far doesn't have any long running functions it really doesn't matter if you were to keep it out of onload, but if you did, you could have a spinner display immediately, and then replace that as soon as you are done loading data to render to the page

Reply:
> Move this code into your onload function.
> 
> So the issue with your list items is you are attempting to create the element and immediately set a value on it hoping that just the element you created gets set to itemX, however what really happens is itemX gets set to the string you are setting to innerText.
> 
> ```
> const item = document.createelement('li');
> item.innerText = 'Hello World';
> ```

Oh so it is setting item1 as a string but....than is it a <li> first and then I am immediately over writing it to a string?

Reply:
so the left most equals will equal what is returned from the operation on the right side of that equals, so document.createElement('li') created that element, but .innerText = 'test' set the property innerText and returned the string which then was set to the item variable. your li was still created but you have no access to it because it was never set to a variable